Xext/panoramiXh.h

```diff
@@ -1,8 +1,10 @@
-
 /*
  *	Server dispatcher function replacements
  */
 
+#ifndef XSERVER_PANORAMIXH_H
+#define XSERVER_PANORAMIXH_H
```

Comment:
@metux, I suspect that I may not have understood you correctly :) so ready to fix the names of the new macros where necessary.